apps/src/lab2/views/components/Instructions/ResourcePanel/index.tsx

```diff
@@ -71,13 +84,21 @@ const ResourcePanel: React.FC<ResourcePanelProps> = ({
   rightHeaderContent,
   includeFooterSpacing = true,
   settings,
+  versionHistoryProps,
   ...instructionsProps
 }) => {
   const {theme} = useTheme();
   const {showRubric} = useRubric();
   const [currentTab, setCurrentTab] = useState<Tabs>(Tabs.Instructions);
   const [isSettingsOpen, setIsSettingsOpen] = useState(false);
   const isUserTeacher = useAppSelector(state => state.currentUser.isTeacher);
+  const [selectedVersion, setSelectedVersion] = useState<string>('');
+  const isViewingOldVersion = useAppSelector(
+    state => state.lab2Project.viewingOldVersion
+  );
+  const viewAsUserId = useAppSelector(state => state.progress.viewAsUserId);
+  const isReadOnly = useAppSelector(isReadOnlyWorkspace);
+  const isWidgetView = instructionsProps.levelProperties.widgetView || false;
 
   const levelId = instructionsProps.levelProperties.id;
```

Comment:
I see that version history panel stays open when a user switches level. It seems like maybe it makes sense to switch back to instructions tab when level id changes?